tachyon/zk/plonk/circuit/assembly.h

```diff
@@ -23,8 +23,8 @@ template <typename PCSTy>
 class Assembly : public Assignment<typename PCSTy::Field> {
  public:
   using F = typename PCSTy::Field;
-  using DensePoly =
-      math::UnivariateDensePolynomial<math::RationalField<F>, PCSTy::kMaxSize>;
+  using DensePoly = math::UnivariateDensePolynomial<math::RationalField<F>,
+                                                    PCSTy::kMaxDegree>;
```

Comment:
A `fixeds_`'s type is evaluations of rational field (not a dense poly!)
So, `DensePoly` should be fixed into `Evals` or `RationalEvals` or ...?